modin/core/execution/unidist/implementations/pandas_on_unidist/partitioning/partition.py

```diff
@@ -162,8 +162,8 @@ def wait(self):
         wait([self._data])
 
     # If unidist has not been initialized yet by Modin,
-    # unidist itself handles initialization when calling `unidist.put`.
-    _iloc = unidist.put(PandasDataframePartition._iloc)
+    # unidist itself handles initialization when calling `UnidistWrapper.put`.
```

Comment:
```suggestion
    # unidist itself handles initialization when calling `unidist.put`.
```
unidist doesn't know anything about UnidistWrapper.

Also, let's add a comment similar to one above for Ray.

Reply:
done